web/package.json

```diff
@@ -38,7 +38,8 @@
       "source": [
         "src/themes/default.css",
         "src/themes/stencila.css",
-        "src/views/vscode.ts"
+        "src/views/vscode.ts",
+        "src/views/vscode-chat.ts"
```

Comment:
Is it best to have to separate files? (I suspect it is but just checking). If so should rename to `vscode-preview.ts` and `vscode-model-chat.ts`

Reply:
Fair enough. I don't think there is a strong need to DRY in the near future. I generally use the [Rule of Three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)).